### DISCUSSION TITLE
Home assistant su Docker non trovo componenti aggiuntivi

### DISCUSSION THREAD

**Discussion by paky1969** [indomus/forum#2985] [GENERALE - Discussioni generiche]
Buongiorno, ho appena installato home assistant su Pi5 su Docker, ma su impostazioni trovo le varie icone tranne quella di componenti aggiuntivi.
Ho anche selezionato modalit'a avanzata. Qualcuno e a conoscenza del problema ?
Grazie

**Comment by snakuzzo** [comment]
Se avessi letto bene cosa andavi ad installare (https://indomus.it/focus/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/) non ti troveresti con questo dubbio.
Non è un problema..questo tipo di installazione non prevede componenti aggiuntivi. Leggi il link che ti ho appena postato.

**Reply by paky1969** [reply]
Grazie mille, non avevo fatto caso a questa particolarita. Ma a questo punto come faccio a installare i componenti aggiuntivi, c'e qualche manuale che ne parla?

**Reply by snakuzzo** [reply]
Ripeto...non esistono componenti aggiuntivi con questo tipo di installazione. Dovresti creare dei container docker a parte per ogni cosa che ti serve. Qualcosa lo trovi sicuro tra le nostre guide, di sicuro ci sono le cose più importanti. Per il resto ti metti e cerchi con google un'immagine docker e relativa documentazione su come istanziare i container

**Reply by paky1969** [reply]
ops...una cosa un po- lunga. Ma se volessi reinstallare HA versione completa di componenti aggiuntivi con docker e possibile? A me serve che il raspberry posso utilizzarlo anche per altro...  Grazie mille

**Reply by snakuzzo** [reply]
Però se fai ste domande, vuol dire che non leggi :)
Puoi provare HA Supervised ... ma la sconsigliamo vivamente perché sarebbe troppo vincolata ai pacchetti sul sistema operativo. Basta un minimo aggiornamento di versione per renderla non più supportata e/o inconsistente.
Il consiglio, in questo caso, è creare una VM per HA OS e poi sotto usi il sistema operativo che ti pare

**Reply by paky1969** [reply]
intanto grazie per i consigli. Ho visto che una parte degli apparati che ho li vede, mi manca ad esempio l-allarme che e un paradox , da qualche parte c e scritto come posso aggiungere questi componenti manualmente?

**Reply by paky1969** [reply]
Buonasera Snakuzzo, ad esempio se seguo questo tutorial LINK RIMOSSO con la versione Home assistant completa di Componenti aggiuntivi, riesco a vedere anche Paradox, ma poi non posso utilizzare il raspberry per altre cose. C-[ modo di installare questi componenti aggiuntivi manualmente e configurarli? Mi basta avere un video tutorial su qualcosa di simile ...se esiste. Grazie ancora

**Reply by snakuzzo** [reply]
Ti ho già risposto in precedenza. Rileggi bene perché la domanda che hai fatto, alla luce di quello che ti ho detto, non ha alcun senso
